src/vocabularyChecker/detectWrongSyntax.ts

```diff
@@ -0,0 +1,79 @@
+import json from "./AllowedVocab.json"
+
+/**
+ * Checks, if all words used, are allowed.
+ * 
+ * 
+ * 
+ */
+
+function preprocessingText(text: string){
+    var cleantext = eliminatePunctuation(text);
+    var textarray = transformTextToStringArray(cleantext);
+    return textarray;
+}
+
+export function checkInput(text: string){
+    let preprocessedText : string[] = preprocessingText(text);
+    let issue : string[] = []; //collect all wrong words in a stringarray
+    for (const word of preprocessedText) {
+        if (!json.foo.includes(word)) {//word not included in AllowedVocab.json
+            console.log(word + " ist kein erlaubtes Wort. Bitte verwenden Sie ein anderes!"); //for debugging purposes. The user should get a message on screen telling him what's wrong.
+            issue.push(word);
+        }
+    }
+    let allWrongWords :string = "";
+    allWrongWords += issue.join //Join all wrong words to one String
+    allWrongWords += " sind nicht erlaubte Wörter. Bitte verwenden Sie andere!"
+    console.log(allWrongWords); //Tell the User what's wrong
+    return issue;
+}
+
+/**
+ * Delete any unwanted punctuation
```

Comment:
"Delete any unwanted punctuation" basically says the same as the method name, "eliminate punctuation". A more helpful documentation would be something like "Turns all unwanted punctuation (some unicode characters, as well as specific punctuation signs) into whitespace".